package.json

```diff
@@ -63,14 +63,14 @@
 		"build-client-both": "CONCURRENT_BUILDS=2 concurrently -c cyan -n \"fallback ,evergreen\" \"npm run build-client-fallback\" \"npm run build-client-evergreen\"",
 		"build-client-if-prod": "node -e \"process.env.CALYPSO_ENV === 'production' && process.exit(1)\" || cross-env-shell NODE_ARGS=${NODE_ARGS:---max_old_space_size=8192} npm run build-client-both",
 		"build-client-if-desktop": "node -e \"(process.env.CALYPSO_ENV === 'desktop' || process.env.CALYPSO_ENV === 'desktop-development') && process.exit(1)\" || cross-env-shell NODE_ARGS=${NODE_ARGS:---max_old_space_size=8192} npm run build-client-fallback",
-		"build-packages": "npx lerna run prepare --stream",
+		"build-packages": "npx lerna run prepare --stream $([ -z \"$CIRCLECI\" ] || echo --concurrency=2)",
```

Comment:
Lowering concurrency is necessary for Jest. Anecdotally, this took 32s vs 20s on latest master build. I'll drop this change.